### PR TITLE
Require all entries listed in a packages bower.json main array

### LINKED ISSUE #39: Only first file of "main" array is included
We are trying to use angular-strap in our project using debowerify. It has template files defined in separate file "dist/angular-strap.tpl.min.js", defined in main section of the bower.json (https://github.com/mgcrea/angular-strap/blob/master/bower.json) but only first file "dist/angular-strap.min.js" gets included.

Is this a correct behavior? If it is what would be the correct way to get template file included in our app?

### PR DESCRIPTION
**Pull request by microadam**
Fixes #39

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -109,30 +109,42 @@ module.exports = function (file, options) {
...
           // if 'main' wasn't specified by this component, let's try
           // guessing that the main file is moduleName.js
-          requiredFilePath = moduleName + '.js';
+          requiredFilePaths = [ moduleName + '.js' ];
         }
+      } else {
+        requiredFilePaths = [ requiredFilePaths ]
       }
 
+      var realtiveRequiredFilePaths = []
```
Comment (maxnordlund) [inline index.js]: Shouldn't this be `relativeRequiredFilePaths`? (Spelling)

Comment (microadam) [inline index.js]: Good spot, Thanks! Fixed

**Comment by eugeneware** [comment]
Thanks @microadam for the PR. Merged and released as 1.3.0.

As per #53 I've given you commit rights as debowerify is an Open Open Source project.

Thanks heaps!

**Comment by bobthecow** [comment]
This is a BC break and will require a new major version number.

File: index.js [hunk 2]
```diff
@@ -109,30 +109,42 @@ module.exports = function (file, options) {
...
-    function replaceNode(node, s) {
+    function replaceNode(node, paths) {
+      var s = paths.shift()
       chunks[node.range[0]] = s;
       for (var i = node.range[0] + 1; i < node.range[1]; i++) {
           chunks[i] = '';
       }
+      paths.forEach(function (p) {
+        var st = '\nrequire(' + p + ')'
+        chunks[node.range[1] + 1] = st
```
Comment (bobthecow) [inline index.js]: does this work for more than two?

Comment (microadam) [inline index.js]: good spot! Fixed in #67 with updated unit test

**Comment by microadam** [comment]
Cheers Eugene :)

**Pull request merged**